docs/architecture/fabric.md

```diff
@@ -39,7 +39,7 @@ underlay network.
 ## Overlay Network
 
 The overlay network runs on top the underlay network to create a virtual network. The overlay network isolates control
-and data plane traffic between different virtual networks and the underlay network. Vitualization is achieved in the
+and data plane traffic between different virtual networks and the underlay network. Visualization is achieved in the
```

Comment:
Visualization?

Reply:
Not sure what was intended here :) I will fix it separately.